cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,133 @@
+<%
+var groupAccess = xblockInfo.get('group_access');
+var selectablePartitions = [];
+var groupIdToName = {};
+
+contentGroups.forEach(function (partition) {
+    if (partition['scheme'] != 'cohort' && partition['scheme'] != 'enrollment_track') {
+        return;
+    }
+    if (partition['scheme'] == 'enrollment_track') {
+        var tracks = [];
+        var trackSelected = false;
+        partition['groups'].forEach(function (group) {
+            if (group['selected']) {
+                trackSelected = true;
+            }
+        });
+        // Only add enrollment tracks if there are multiple tracks to restrict from
+        if (partition['groups'].length < 2 && !trackSelected) {
+            return;
+        }
+    }
+    selectablePartitions.push(partition);
+});
+
+selectablePartitions.forEach(function (partition) {
+    var schemeId = partition['id'];
+    var groups = partition['groups'];
+    groups.forEach(function (group) {
+        var groupId = group['id'];
+        var groupName = group['name'];
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        groupIdToName[fullId] = groupName;
+    })
+});
+
+var deletedGroups = [];
+
+for (var schemeId in groupAccess) {
+    groupAccess[schemeId].forEach(function (groupId) {
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        if (!groupIdToName[fullId]) {
+            deletedGroups.push(fullId);
+        }
+    })
+}
+
+// Need to include deleted groups logic to not change is_active boolean in contentstore/utils.py get_user_partition_info
```

Comment:
@jlajoie Thanks for investigating. If you can fix this bug in < 30 minutes, do so. Otherwise create a bug for it (with your excellent reproduction steps!) and move on.